org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/context/RunContext.java

```diff
@@ -571,6 +576,10 @@ protected void fillEmpty() {
 
   /**
    * Creates a copy of <code>this</code> context.
+   * <p>
+   * <b>NOTE</b> use {@link RunContexts#copyCurrent()} when creating a nested RunContext inside a running chain. This
```

Comment:
Maybe we should explain the use case of this method a bit more in detail and clarify that it should not be used under normal circumstances.

Something like this:

`<b>Important:</b> Be careful when using this method! It should only be used to create a shallow copy of a run context that you have under control, e.g. a template context. Avoid using it to create a copy of the current run context, as simply copying may not make sense for all values (e.g. transaction scope). In most cases, it is recommended to use {@link RunContext#copyCurrent()} instead to create a new run context. Unlike {@link copy()}, it initializes the values according to the current thread state. For example, the transaction scope is reset to the default value of the active {@link RunContextFactory} and the transaction is taken from {@code ITransaction.CURRENT.get()}.`

Reply:
thank you, very good point, done.